packages/p2p/src/components/dp2p-blocked/dp2p-blocked.tsx

```diff
@@ -1,12 +1,12 @@
 import * as React from 'react';
 import { Icon, Text } from '@deriv/components';
-import Dp2pBlockedChecklist from './dp2p-blocked-checklist.jsx';
-import Dp2pBlockedDescription from './dp2p-blocked-description.jsx';
+import Dp2pBlockedChecklist from './dp2p-blocked-checklist';
+import Dp2pBlockedDescription from './dp2p-blocked-description';
 import { Localize } from '../i18next';
```

Comment:
should we change the import here? `to import { Localize } from 'Components/i18next';`